core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -200,6 +200,17 @@ default ConnectorTableSchema getTableSchema(ConnectorSession session, ConnectorT
         return getTableMetadata(session, table).getTableSchema();
     }
 
+    /**
+     * Return schema table name for the specified table handle.
+     * This method is useful when getting full table metadata is expensive.
```

Comment:
`getTableSchema` was introduced to be a lighter weight alternative to `getTableMetadata`
do we need yet another "even lighter" method?

Reply:
(for sure "full table metadata"  needs update here)

Reply:
Updated the comment. I wanted to avoid another helper method, but other approach also require SPI change either in my understanding.